augur/__init__.py

```diff
@@ -76,6 +78,15 @@ def run(argv):
     except RecursionError:
         print("FATAL: Maximum recursion depth reached. You can set the env variable AUGUR_RECURSION_LIMIT to adjust this (current limit: {})".format(sys.getrecursionlimit()))
         sys.exit(2)
+    except Exception:
+        traceback.print_exc()
+        print("\n")
+        print(dedent("""\
+            An error occurred (see above) that has not been properly handled by Augur.
+            To report this, please open a new issue including the original command and the error above:
+                <https://github.com/nextstrain/augur/issues/new/choose>
```

Comment:
> @tsibley: I understand that it's a bug from our own perspective in the sense that we should handle FileNotFoundError, but it's not a bug from the user's perspective: they've just mis-specified the file or something and the fix is on them not on us.

I think this is inevitable - some exceptions have a clear error message so they've been passed through to the user as a "feature". However, you're also right that this is a bug from our own perspective (#642) and I've rephrased the message here to hopefully reflect this better.

Since you've described the uncaught FileNotFoundError here, I've addressed it in c25db932e85834ba1db7c75984f7815b0ae3a06d.